build/istanbul-config.js

```diff
@@ -1,24 +1,20 @@
 'use strict';
 
-var path = require('path');
+var unitTestCoverageDirectory = './coverage/unit';
+var unitTestCoverageReportHtmlFile = './coverage/unit/index.html';
 
-var unitTestCoverageDirectory = path.resolve('./coverage/unit/');
-var unitTestCoverageReportHtmlFile = path.resolve('./coverage/unit/index.html');
-
```

Comment:
Was the thinking here that the comments were redundant? I put them in there because I was previously unaware that we could apply thresholds to each file as well as at an aggregate, but if the var names are explanatory enough then fair enough